ert3_examples/snake_oil/run_demo

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# This script serves both as an example, and for CI purposes
+
+set -e
+
+ert3 clean --all  # Only relevant when run manually
+
+# Let ERT draw and run a stochastic ensemble:
+ert3 run evaluation
+ert3 export evaluation
+# Assert that we have FOPT in the output
+grep FOPT experiments/evaluation/data.json >/dev/null
+
+
+# DOE (design of experiment)
+design2params 0 resources/designmatrix.xlsx designsheet defaults  # [semeio]
+python designmatrixtxt2json.py  # [hack]
+ert3 record load BPR_138_PERSISTENCE resources/designed_bpr_138_persistence.json
```

Comment:
I think we should provide Transformation that loads either multiple `NumericalRecord` / files (matching a certain filer eg. designed_*) or a single file with multiple records within into a single `RecordTree`...

Reply:
Agree, but it is a bigger scope than this PR.